packages/analytics/dashboard-renderer/src/components/AnalyticsChartRenderer.vue

```diff
@@ -24,6 +26,7 @@ const props = defineProps<RendererProps<BarChartOptions>>()
 const chartTypeLookup = {
```

Comment:
I can't comment directly on the line (yay Github), but I think on line 24 the props should be `RendererProps<BarChartOptions | TimeseriesChartOptions>`.  If this causes any trouble, then don't worry about fixing it, but please let me know so I can try to fix it later.